phpstan-baseline.neon

```diff
@@ -59,3 +59,9 @@ parameters:
 			identifier: argument.type
 			count: 1
 			path: src/Persistence/Search/Query/ListFacetQueryBuilder.php
+
+		-
+			message: '#^Cannot call method getFullURL\(\) on MediaWiki\\Title\\Title\|null\.$#'
```

Comment:
I ignored this for now, since "Special:Search" should exist. If there's some wiki-specific config that can make the page disappear, then we probably need to make the whole sentence dynamic or find the search page in a different way.